src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -238,7 +235,7 @@ public String getUserAnalysisOutputsPage(Model model) {
 	 * @return redirect
 	 */
 
-	@RequestMapping(value = "/{submissionId}*")
+	@RequestMapping(value = "/{submissionId}**")
```

Comment:
Can you break the ajax requests out into a new `AnalysisAjaxController`?

Reply:
This file is so massive, we need to look at ways to simplify it.  Maybe some of this should be handled at the service layer?